XYZDrivers/src/main/java/com/xyzdrivers/controllers/AuthController.java

```diff
@@ -70,7 +63,47 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
     @Override
     protected void doPost(HttpServletRequest request, HttpServletResponse response)
             throws ServletException, IOException {
-        processRequest(request, response);
+        
+        String username = (String)request.getParameter("username");
+        String password = (String)request.getParameter("password");
+        String userType = (String)request.getParameter("userType");
+        
+        User loggedInUser = null;
+        HttpSession session = request.getSession(true);
+                
+        try {
+            if (userService.checkLoginDetails(username, password))
```

Comment:
Please be consistent with the code formatting. Seems that { starts on a new line and sometimes doesn't